src/middlewares/userValidations.js

```diff
@@ -16,7 +16,7 @@ const validateUser = path => (req, res, next) => {
   if (_.has(Schemas, path)) {
     const schema = _.get(Schemas, path);
     const response = Joi.validate(user, schema, { abortEarly: false });
-    const errors = Helper.buildErrorResponse(response);
+    const errors = Helper.handleValidation(response);
```

Comment:
You should discard the changes in this file

Reply:
Done!